### DISCUSSION TITLE
Someone's Better Hud

### DISCUSSION THREAD

**Discussion by SomeoneNKnuckles** [SomeoneNKnuckles/Someone-s-Mods#3] [General]
![Better Hud](https://github.com/user-attachments/assets/9fe64e66-ab7b-4953-ba4b-8ee4b040debd) ![Better Hud Mania](https://github.com/user-attachments/assets/32c0e209-e923-4ce9-8ed6-d6c213b08fea)

![Other](https://private-user-images.githubusercontent.com/154085372/384704365-79e888f7-c9c4-4cba-acd9-1c483cc3ec5e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEyNTQ5NzcsIm5iZiI6MTczMTI1NDY3NywicGF0aCI6Ii8xNTQwODUzNzIvMzg0NzA0MzY1LTc5ZTg4OGY3LWM5YzQtNGNiYS1hY2Q5LTFjNDgzY2MzZWM1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExMFQxNjA0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYjM2MTdhMjU4YmVjYWRmYWMyMzJjMTZiMjNjZGFmYzlhYjgxMDUwOTFjZDgxOWU2MGU2ZWU2N2VjZGYyODQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.8mhunfAHlnGOKOkaA8PmYmCLl1TcR-u4oagGvNkLOZ8) What Extra Slots Icons Should I Add?
![Other](https://github.com/user-attachments/assets/28e56e60-4e1b-4cdb-8412-ba3d5b75d35a) What Compability Should I Add?

**Comment by Mazher94** [comment]
You should add compatibility with ES pana.
I think I'll be cool to have!

**Comment by Mazher94** [comment]
Actually, I have a question. Is there seperate live huds for Super and hyper forms? (Like dedicated super icon?)

**Reply by SomeoneNKnuckles** [reply]
![image](https://github.com/user-attachments/assets/c4100929-3d53-47ab-97a5-c9885c75992a)
yes

**Reply by Mazher94** [reply]
Well, they don't seem to work for me for some reason.

Actually, I looked into the files and couldn't find any of those sprites, or the json script.

**Reply by SomeoneNKnuckles** [reply]
in the next update.
